src/components/scenes/ChangePasswordScene.tsx

```diff
@@ -177,23 +172,14 @@ const ChangePasswordSceneComponent = ({
             maxLength={100}
           />
         </EdgeAnim>
-        <EdgeAnim
-          style={styles.actions}
-          enter={{ type: 'fadeInDown', distance: 50 }}
-          exit={{ type: 'fadeOutDown', distance: 50 }}
-        >
-          {spinning ? (
-            <MainButton disabled marginRem={0.5} type="primary" spinner />
-          ) : (
-            <MainButton
-              label={mainButtonLabel}
-              disabled={isNextButtonDisabled || confirmPassword === ''}
-              marginRem={0.5}
-              onPress={handleNext}
-              type="primary"
-            />
-          )}
-        </EdgeAnim>
+        <SceneButtons
+          primary={{
+            label: mainButtonLabel,
+            disabled: isNextButtonDisabled || confirmPassword === '',
+            onPress: handleNext
```

Comment:
So if you press "enter" on the keyboard, the button won't spin. We still need the `spinning` / `setSpinning` state to handle this case.